src/sage/graphs/generic_graph.py

```diff
@@ -1900,6 +1900,68 @@ def to_dictionary(self, edge_labels=False, multiple_edges=False):
 
         return d
 
+    def _vertex_indices_and_keys(self, vertices=None, *, sort=None):
+        r"""
+        Process a ``vertices`` parameter.
+
```

Comment:
May be you could add that this is a helper function for methods like `adjacency_matrix`, `kirchhoff_matrix`, etc.

Reply:
Good idea, done in 98323cb719e